src/apps/common/submodules/mediaSelect/mediaSelect.js

```diff
@@ -113,32 +120,44 @@ define(function(require) {
 			} else if (skin === 'tabs') {
 				template = $(self.getTemplate({
 					name: 'tabs-layout',
-					data: formattedData,
+					data: _.merge(formattedData, {
```

Comment:
It is better to merge values into a new object, in order to not to modify the original one that has been passed as a parameter, to avoid any unexpected side effects in the callers (unless it is the intention).
```suggestion
					data: _.merge({}, formattedData, {
```